snapcraft/internal/log.py

```diff
@@ -75,6 +76,9 @@ def configure(logger_name=None, log_level=None):
     # INFO by default for the requests lib as it is too noisy
     if log_level == logging.DEBUG:
         logging.getLogger("requests").setLevel(log_level)
+
+        # Debug HTTP connections.
+        HTTPConnection.debuglevel = 1
```

Comment:
Does this show macaroons?

Reply:
Almost certainly, hrmm...

Reply:
Maybe we should take this into account for the revamping of debug logging.  Have some conditional with a warning..